src/cityreader/cityreader.py

```diff
@@ -16,11 +17,28 @@
 # should not be loaded into a City object.
 cities = []
 
+class City:
+  def __init__(self, city, lat, lon):
+    self.name = city
+    self.lat = lat
+    self.lon = lon
+  def __str__(self):
+    return f"{self.name}, {self.lat}, {self.lon}"
+
 def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
+
+  with open('cities.csv') as csvfile:
+    reader = csv.DictReader(csvfile)
+    for row in reader:
+      name = row['city']
+      lat = float(row['lat'])
+      lon = float(row['lng'])
+      cities.append(City(name, lat, lon))
+
+
     return cities
 
 cityreader(cities)
```

Comment:
list of similar objects would be like
 cities = ["Vegas", "Philly", "LA"]
city = cities[0]